packages/app/src/app/pages/Dashboard/Components/Sandbox/SandboxBadge.tsx

```diff
@@ -0,0 +1,36 @@
+import { Icon, Stack, Text } from '@codesandbox/components';
+import React from 'react';
+
+export interface SandboxBadgeProps {
+  isDevbox: boolean;
+  isTemplate: boolean;
+  restricted: boolean;
+}
+
+export const SandboxBadge: React.FC<SandboxBadgeProps> = ({
+  isDevbox,
+  isTemplate,
+  restricted,
+}) => {
+  const boxIcon = isDevbox ? 'boxDevbox' : 'boxSandbox';
+  let boxTypeColor = isDevbox ? '#FFFFFF' : '#A6A6A6';
+  let boxTypeLabel = isDevbox ? 'Devbox' : 'Sandbox';
+
+  if (restricted) {
+    boxTypeColor = '#F7CC66';
+    boxTypeLabel = 'Restricted';
+  }
+
+  if (isTemplate) {
+    boxTypeLabel = 'Template';
```

Comment:
I think we could compound the template label with the project type, as I did on the editor:
![Screenshot 2024-01-03 at 09 43 04](https://github.com/codesandbox/codesandbox-client/assets/4838076/f61943dd-9f24-4f6e-b5eb-3dcd0cf91b8e)

Do you think there is room for a longer label?

Reply:
that's a good idea, I like it!